src/common/AdaptiveDecrypter.h

```diff
@@ -8,25 +8,18 @@
 
 #pragma once
 
-#include "../CryptoMode.h"
-
 #include <stdexcept>
 #include <string_view>
 
 #include <bento4/Ap4.h>
 
+struct CryptoInfo;
```

Comment:
please add a comment above
`// Forward`

Reply:
take in account that if you add a forward other files that will include this header must add also the header that include CryptoInfo struct